config/initializers/devise.rb

```diff
@@ -168,7 +168,7 @@
   # :time  = Re-enables login after a certain amount of time (see :unlock_in below)
   # :both  = Enables both strategies
   # :none  = No unlock strategy. You should handle unlocking by yourself.
-  config.unlock_strategy = :time
+  config.unlock_strategy = :both
```

Comment:
Hm, I think a manual unlocking strategy would be best. If you can write an issue with the unlock workflow, that would be awesome.